org.openhantek.OpenHantek6022.yml

```diff
@@ -7,16 +7,14 @@ command: OpenHantek
 finish-args:
   # allow displaying on x11 and wayland
   - --share=ipc
-  - --socket=x11
+  - --socket=fallback-x11
   - --socket=wayland
   # needed to get access to scope on usb
   - --device=all
   # needed otherwise you get weird graphical glitches on wayland
   - --env=QT_QPA_PLATFORM=xcb
```

Comment:
You should have removed the wayland socked instead since here you force the use of xcb QPA.